app/styleguide/components/s-theme-selector.md

```diff
@@ -0,0 +1,6 @@
+### s-theme-selector
+
+#### Theme changer:
```

Comment:
- [ ] The titles are not needed. It should be clear from the automatically created titles.